behave/README.md

```diff
@@ -5,6 +5,9 @@ This readme assumes that the working directory is that of this document, which i
 root of this repository: the root of this repository now provides a means to deliver and use the
 functionality in this directory.
 
+Usage: To use a new set of tests from this repo, the target repo must set E2E_BRANCH=<branch name of this repo>
```

Comment:
I don't think this is the best place to put this in the readme as it doesn't fit into the structure, feels out of place and a little random. I would put this under the header section of `Test file structure` as it feel related.

Here an example, feel free to tweak the wording if it's not totally right.
```
## Testing branches in Circleci

When you need to test a targeted branch in CircleCi, you will need to add 
`echo "export E2E_BRANCH= >> $BASH_ENV` within your targeted branch projects CircleCi config. This will need to be applied under your projects CircleCi config `behave` values
```